src/helpers/translations.helper.ts

```diff
@@ -2,7 +2,14 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 import i18n from "i18next";
+import { initReactI18next } from "react-i18next";
 import { ResourceName } from "../assets/translations/types";
+import resources from "../assets/translations";
+
+i18n.use(initReactI18next).init({
+  fallbackLng: "pl",
+  resources,
+});
```

Comment:
Przeniosłem tutaj, ponieważ miałem sytuację w której metoda `t` była używana zanim były ustawione resursy.
Nie wygląda na to że to coś psuję, więc nie zastanawiałem się nad tym dłużej. 
Ale jeśli macie do tego jakieś uwagi - to poprawię